src/index.js

```diff
@@ -113,6 +113,17 @@ class Service {
     return removeBulk(this, params)
       .catch(errorHandler);
   }
+
+  raw (query) {
```

Comment:
Could we keep the parameter names consistent with other functions? `params` instead of `query` here.